<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>755</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.SF NS Text</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0,0" columnstretch="1,1">
    <item row="2" column="0">
     <widget class="QGroupBox" name="filterGroupBox">
      <property name="title">
       <string>Filter</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="attrFilterGroupBox">
         <property name="title">
          <string>Attributes</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="hiddenFilterCheckBox">
            <property name="text">
             <string>Hidden</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="systemFilterCheckBox">
            <property name="text">
             <string>System</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fileDirFilterGroupBox">
         <property name="title">
          <string>File / Dir</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="filesFilterCheckBox">
            <property name="text">
             <string>Files</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dirsFilterCheckBox">
            <property name="text">
             <string>Dirs</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="nameMaskFilterGroupBox">
         <property name="title">
          <string>Name mask</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="nameMaskFilterLineEdit">
            <property name="text">
             <string>*</string>
            </property>
            <property name="placeholderText">
             <string>*.txt d???.jpg</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableView" name="folderView">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="currentPathLineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="sortGroupBox">
      <property name="title">
       <string>Sort</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="sortTypeGroupBox">
         <property name="title">
          <string>Sort type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="nameSortRadioButton">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="extSortRadioButton">
            <property name="text">
             <string>Extention</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sizeSortRadioButton">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="lastModifiedSortRadioButton">
            <property name="text">
             <string>Last modified</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="foldersGroupBox">
         <property name="title">
          <string>Folders</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="firstFoldersRadioButton">
            <property name="text">
             <string>First</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="lastFoldersRadioButton">
            <property name="text">
             <string>Last</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="NoSpecifyRadioButton">
            <property name="text">
             <string>No specify</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QGroupBox" name="sortType2ndGroupBox">
         <property name="title">
          <string>Sort type(secondary)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="nameSort2ndRadioButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="extSort2ndRadioButton">
            <property name="text">
             <string>Extention</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sizeSort2ndRadioButton">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="lastModifiedSort2ndRadioButton">
            <property name="text">
             <string>Last modified</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="noneSort2ndRadioButton">
            <property name="text">
             <string>None</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" rowspan="3">
        <widget class="QGroupBox" name="orderGroupBox">
         <property name="title">
          <string>Order</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="ascendingRadioButton">
            <property name="text">
             <string>Ascending</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="decendingRadioButton">
            <property name="text">
             <string>Decending</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" rowspan="3">
        <widget class="QGroupBox" name="caseSensitiveGroupBox">
         <property name="title">
          <string>Case sensitive</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="sensitiveRadioButton">
            <property name="text">
             <string>Sensitive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="InsensitiveRadioButton">
            <property name="text">
             <string>Insensitive</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="formatGroupBox">
      <property name="title">
       <string>Single pane(&amp;S)</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
       <item>
        <widget class="QGroupBox" name="formatFileSizeGroupBox">
         <property name="title">
          <string>File size</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_21" columnstretch="0,1,0">
          <item row="3" column="1" colspan="2">
           <widget class="QCheckBox" name="formatFileSizeCommaCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Comma</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QRadioButton" name="formatFileSizeSIRadioButton">
            <property name="text">
             <string>SI format(KB, MB, GB, ...)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>12</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QRadioButton" name="formatFileSizeDetailRadioButton">
            <property name="text">
             <string>Detail</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QRadioButton" name="formatFileSizeIecRadioButton">
            <property name="text">
             <string>IEC format(KiB, MiB, GiB, ...)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="formatDateGroupBox">
         <property name="title">
          <string>Date</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_22" columnstretch="0,1,0" columnminimumwidth="0,0,0">
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="formatDateOriginalLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>yyyy-MM-dd HH:mm:ss</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QRadioButton" name="formatDateDefaultRadioButton">
            <property name="text">
             <string>Default</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>12</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QRadioButton" name="formatDateISORadioButton">
            <property name="text">
             <string>ISO 8601</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QRadioButton" name="formatDateOriginalRadioButton">
            <property name="text">
             <string>Original</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
